### ISSUE TITLE
Update documentation to suggest loading the python module

### ISSUE THREAD

**Issue by half-adder** [mckaylabunc/cutNrun-pipeline#3]
The python module needs to be loaded on longleaf before `sh slurmSubmission.sh` is run. The documentation should reflect this.

**Comment by jemcpher** [comment]
agreed. just ran into this problem helping Megan get started with the pipeline. `module load python/3.6.6` does the trick.
